<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
background-color: #32383D;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="btns_container">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
}

QPushButton {
color: #A1B4C4;
border-radius: 5px;

background: qlineargradient(x1:0, y1:0, x1:0, y2:1,
                                  stop:0 #dbddde, stop:0.05 #6f767c, stop:1  #4C555C);
height: 25px;
font: 12pt &quot;Segoe UI&quot;;
}

QPushButton:hover {
background: qlineargradient(x1:0, y1:0, x1:0, y2:1,
                                  stop:0 #e8f9f7, stop:0.1 #a3e8e1, stop:0.9  #1BC6B4, stop:1 #083b36);
color: #034159;
border-top-left-radius: 15px;
border-top-right-radius: 15px;
height: 40px;
font: 750 12pt &quot;Segoe UI&quot;;
}

</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btn_copy">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Copy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_cut">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Cut</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_past">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Paste</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_undo">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Undo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_redo">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Redo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_set_plain_text">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Set plain text</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_set_html">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Set HTML</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_clear">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>btn_copy</zorder>
      <zorder>btn_undo</zorder>
      <zorder>btn_past</zorder>
      <zorder>btn_set_plain_text</zorder>
      <zorder>btn_cut</zorder>
      <zorder>btn_set_html</zorder>
      <zorder>btn_redo</zorder>
      <zorder>btn_clear</zorder>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="text_container">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextEdit {
background-color: #202529;
color: #A1B4C4;
selection-background-color: #A1B4C4;
selection-color: #202529;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextEdit" name="text_edit">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_copy</sender>
   <signal>clicked()</signal>
   <receiver>text_edit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_cut</sender>
   <signal>clicked()</signal>
   <receiver>text_edit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_past</sender>
   <signal>clicked()</signal>
   <receiver>text_edit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_undo</sender>
   <signal>clicked()</signal>
   <receiver>text_edit</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>479</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>484</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_redo</sender>
   <signal>clicked()</signal>
   <receiver>text_edit</receiver>
   <slot>redo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>592</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>596</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_clear</sender>
   <signal>clicked()</signal>
   <receiver>text_edit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>941</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>946</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
